pkg/scheduler/predicates.go

```diff
@@ -26,28 +26,28 @@ import (
 )
 
 type NodeInfo interface {
-	GetNodeInfo(nodeID string) (*api.Node, error)
+	GetNodeInfo(node api.Node) (*api.Node, error)
```

Comment:
GetNodeInfo is responsible for transforming nodeID into the internal cache - that's a detail that is not part of this call.